free.js

```diff
@@ -34,13 +34,22 @@ Free.prototype.chain = function(f) {
 
 const liftF = command => Free.Impure(command, Free.Pure)
 
+const IGNORE_VALUE = typeof Symbol != "undefined"
+  ? Symbol.for("free/ignore_value")
+  : "@@free/ignore_value"
+
```

Comment:
Will symbols cause us some of the same headaches as [`instanceof` checks][1]?


[1]: https://github.com/sanctuary-js/sanctuary/issues/100